README.md

```diff
@@ -26,6 +26,11 @@ Images for other architectures (e.g. `arm`, `arm64`) can be build locally by run
 docker build -t linksmart/hds .
 ```
 
+### Demo mode
+To run Historical Datastore in demo mode (with continuously growing dummy senml data)
+```
+docker run -p 8085:8085  linksmart/hds -demo -conf /conf/docker.json
```

Comment:
`-conf /conf/docker.json` is the default path, already in Dockerfile.

Reply:
It is in dockerfile as `CMD ["-conf", "/conf/docker.json"]`.  This will be overriden by user specific command as mentioned [here](https://docs.docker.com/engine/reference/builder/)
>  If the user specifies arguments to docker run then they will override the default specified in CMD.

I tried myself personally too. default didnt work once I give a command line option